### PR TITLE
LPS-180016, LPS-180017, LPS-180018, LPS-180019, LPS-180020, LPS-180021, LPS-180022, LPS-180023, LPS-180024, LPS-180025 Simplify tests

### PR DESCRIPTION
**Pull request by Tim-Cao**
https://liferay.atlassian.net/browse/LPS-180016
https://liferay.atlassian.net/browse/LPS-180017
https://liferay.atlassian.net/browse/LPS-180018
https://liferay.atlassian.net/browse/LPS-180019
https://liferay.atlassian.net/browse/LPS-180020
https://liferay.atlassian.net/browse/LPS-180021
https://liferay.atlassian.net/browse/LPS-180022
https://liferay.atlassian.net/browse/LPS-180023
https://liferay.atlassian.net/browse/LPS-180024
https://liferay.atlassian.net/browse/LPS-180025

### CODE REVIEW

**Comment by liferay-echo** [comment]
Thank you for submitting this PR. 👏

Make sure it follows the [Echo Pull Request Template](https://gist.github.com/liferay-echo/5a4455285ff1800f9da146816d85e67d#file-pull-request-template-md) if you haven't already done so. 

Learn more about it [here](https://liferay.atlassian.net/wiki/spaces/ENGECHO/pages/2027455014/Echo+Pull+Request+Etiquette).

**Comment by Tim-Cao** [comment]
ci:test:sf

File: modules/apps/headless/headless-delivery/headless-delivery-test/src/testIntegration/java/com/liferay/headless/delivery/resource/v1_0/test/NavigationMenuResourceTest.java [hunk 1]
```diff
@@ -86,127 +86,240 @@ public void setUp() throws Exception {
...
 				testGroup.getGroupId(), TestPropsValues.getUserId()));
+	}
 
-		siteNavigationMenuItem = _addSiteNavigationMenuItem(
-			postNavigationMenu.getId(), FileEntry.class.getName(),
-			UnicodePropertiesBuilder.create(
-				true
-			).put(
-				"className", DLFileEntry.class.getName()
+	private void _assertNavigationMenuItem(
```
Comment (ruben-pulido) [inline modules/apps/headless/headless-delivery/headless-delivery-test/src/testIntegration/java/com/liferay/headless/delivery/resource/v1_0/test/NavigationMenuResourceTest.java:215]: What do you think about creating several overloaded methods, one accepting JournalArticle as parameter, another accepting BlogEntry, ... and calling the method that corresponds from the different tests? 
That way we don't need the instance of checks

**Review by ruben-pulido — CHANGES_REQUESTED** [review]

File: modules/apps/headless/headless-delivery/headless-delivery-test/src/testIntegration/java/com/liferay/headless/delivery/resource/v1_0/test/NavigationMenuResourceTest.java (hunk 1 above)
Comment (Tim-Cao) [reply modules/apps/headless/headless-delivery/headless-delivery-test/src/testIntegration/java/com/liferay/headless/delivery/resource/v1_0/test/NavigationMenuResourceTest.java:215]: For overloaded methods, do you mean to create different methods for different type assets?
@ruben-pulido

**Comment by Tim-Cao** [comment]
ci:test:sf

**Comment by ruben-pulido** [comment]
ci:forward

**Comment by Tim-Cao** [comment]
See https://github.com/brianchandotcom/liferay-portal/pull/137549

**Comment by Tim-Cao** [comment]
ci:close

**Pull request closed without merging**